### ISSUE TITLE
Placeholder utility not working as expected

### ISSUE THREAD

**Issue by owaiswiz** [ben-rogerson/twin.macro#26]
Just came across this yesterday, placeholder utility (ex. `placeholder-red-500`) does not work as expected. They instead apply the color property on the input element directly instead of the ::placeholder  pseudoelement.

Here's a [codesandbox ](https://codesandbox.io/s/twin-placeholder-utility-issue-u4kmw).

(I think it's because how they are defined here ?)
https://github.com/ben-rogerson/twin.macro/blob/27bd97600bd706f70068fa40ed6956dfff5ea6f6/src/config/dynamicStyles.js#L165
I think they would need to be handled as a special case (like how you've done with :before and :after). I tried to find how one would do that but couldn't. 

For now to get around this for other people who see this (that are using a css-in-jss solution), you have to apply styles using the ::placeholder pseudoelement manually
```javascript
const Input = styled.input`
 ${tw`p-4`}
 ::placeholder { 
    ${tw`text-red-500`} 
}
`;
```

**Comment by ben-rogerson** [comment]
Thanks for submitting this bug and for the perfect demo. I'll fix this bug up pronto.

**Comment by ben-rogerson** [comment]
The fix will be in 1.0.0-alpha.9, closing :)

**Comment by ben-rogerson** [comment]
Here's [your demo](https://codesandbox.io/s/twin-placeholder-utility-issue-et1yc?file=/src/pages/index.js) working in the new twin version.
